omero/sysadmins/unix/install-web/install-apache.txt

```diff
@@ -59,4 +89,9 @@ To configure an HTTPS server follow
 `the Apache documentation <http://httpd.apache.org/docs/trunk/mod/mod_ssl.html>`_.
 
 
-Then reload Apache.
+Running OMERO.web (Unix/Linux)
+------------------------------
+
+OMERO.web is used in 'daemon' mode where UNIX sockets are used to communicate
+between the Apache child processes and the daemon processes which are
+to handle a request. Reload Apache.
```

Comment:
Why am I reloading Apache in this section, after a descriptive text that didn't require any configuration changes?

Should this not be placed after the last instruction to change the Apache configuration?